packages/components/rollup.config.mjs

```diff
@@ -9,6 +9,7 @@ export default rollupConfig({
     actionsV3: "./src/__actions__/v3.ts",
     containersV1: "./src/__containers__/v1.ts",
     containersV2: "./src/__containers__/v2.ts",
+    illustrationsV3: "./src/__illustrations__/v3.ts",
```

Comment:
You'll need `illustrationsV1` and `illustrationsV2`

Reply:
We'll do it in a separate PR